posthog/temporal/batch_exports/bigquery_batch_export.py

```diff
@@ -113,6 +166,27 @@ def bigquery_client(inputs: BigQueryInsertInputs):
         client.close()
 
 
+def bigquery_export_default_fields() -> list[BatchExportField]:
+    """Default fields for a BigQuery batch export.
+
+    Starting from the common defualt fields, we add and tweak some fields for
+    backwards compatibility.
+    """
+    batch_export_fields = default_fields()
+    batch_export_fields.append(
+        {
+            "expression": "nullIf(JSONExtractString(properties, '$ip'), '')",
+            "alias": "ip",
+        }
+    )
+    # Fields kept for backwards compatibility with legacy apps schema.
+    batch_export_fields.append({"expression": "toJSONString(elements_chain)", "alias": "elements"})
+    batch_export_fields.append({"expression": "''", "alias": "site_url"})
+    batch_export_fields.append({"expression": "NOW64()", "alias": "bq_ingested_timestamp"})
```

Comment:
Notice that moving `bq_ingested_timestamp` here comes with a bit of a downside: We cannot really test for it anymore, as `NOW64()` is unstable.

Reply:
But it's easier to treat is as any other default field...